modules/oe_theme_helper/tests/src/Kernel/RetinaScaleEffectTest.php

```diff
@@ -113,7 +113,7 @@ public function testTripleMultiplierRetinaScaleEffect(): void {
     $this->assertToolkitOperationsCalled(['scale']);
 
     $calls = $this->imageTestGetAllCalls();
-    $this->assertEquals($this->image->getWidth() * 3, $calls['scale'][0][0], 'Width is triple the original size.');
+    $this->assertEquals($this->image->getWidth() * 3, $calls['scale'][0][0], 'Width is three times the original size.');
```

Comment:
Same as above. Messages should reflect the failure, not the success. Removing them is OK too.